csharp/src/SeedLang/Ast/AstDumper.cs

```diff
@@ -0,0 +1,319 @@
+// Copyright 2021-2022 The SeedV Lab.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Text;
+using SeedLang.Runtime;
+
+namespace SeedLang.Ast {
+  internal static class OperatorExtensions {
+    // Returns the internal string representation of binary operators.
+    internal static string Symbol(this BinaryOperator op) {
+      switch (op) {
+        case BinaryOperator.Add:
+          return "+";
+        case BinaryOperator.Subtract:
+          return "-";
+        case BinaryOperator.Multiply:
+          return "*";
+        case BinaryOperator.Divide:
+          return "/";
+        case BinaryOperator.FloorDivide:
+          return "//";
+        case BinaryOperator.Power:
+          return "**";
+        case BinaryOperator.Modulo:
+          return "%";
+        default:
+          throw new NotImplementedException($"Unsupported binary operator: {op}.");
+      }
+    }
+
+    // Returns the internal string representation of boolean operators.
+    internal static string Symbol(this BooleanOperator op) {
+      switch (op) {
+        case BooleanOperator.And:
```

Comment:
"And", "Or" and "Not: both Python and Lua use lowercase form for those operators - "and", "or", "not". Shall we follow this convention for internal representations of them?

Reply:
Fixed.